docs/src/pages/components/data-grid/accessibility/accessibility.md

```diff
@@ -76,9 +76,10 @@ Use the arrow keys to move the focus.
 
 ### Sorting
 
-|                              Keys | Description          |
-| --------------------------------: | :------------------- |
-| <kbd>CTRL</kbd> + Click on header | Enable multi-sorting |
+|                               Keys | Description          |
+| ---------------------------------: | :------------------- |
+|  <kbd>CTRL</kbd> + Click on header | Enable multi-sorting |
+| <kbd>Shift</kbd> + Click on header | Enable multi-sorting |
```

Comment:
It's going to render "fucked-up" otherwise once we upgrade the docs infra to v5.

```suggestion
|  <kbd class="key">CTRL</kbd> + Click on header | Enable multi-sorting |
| <kbd class="key">Shift</kbd> + Click on header | Enable multi-sorting |
```

<img width="363" alt="Screenshot 2021-04-15 at 23 36 42" src="https://user-images.githubusercontent.com/3165635/114941426-74880200-9e43-11eb-8cd9-9ce45180cc3c.png">

Reply:
Maybe it's a small enough clean-up to be handle here for all the documentation? Not sure